CHANGELOG.md

```diff
@@ -0,0 +1,175 @@
+Change MSI Service Error handling ([#5467](https://github.com/osquery/osquery/pull/5467))
+
+Allow mounting SQLite DBs using WAL journaling with ATC ([#5525](https://github.com/osquery/osquery/issues/5225), [#5633](https://github.com/osquery/osquery/pull/5633))
+
+Fix for mount table interacting with direct autofs. ([#5635](https://github.com/osquery/osquery/pull/5635)) 
+
+Fix HTTP Host Header and port logic ([#5576](https://github.com/osquery/osquery/pull/5576))
+
+windows/certificates: Fix bug in environment variable expansion ([#5697](https://github.com/osquery/osquery/pull/5697))
+
+windows/certificates: Do not filter out system accounts ([#5696](https://github.com/osquery/osquery/pull/5696))
+
+windows/certificates: Improve table's coverage of Personal certificates ([#5640](https://github.com/osquery/osquery/pull/5640))
+
+windows/certificates: Fix enumeration bugs ([#5631](https://github.com/osquery/osquery/pull/5631))
+
+tables: Add optimization back to macOS `users` and `groups` ([#5684](https://github.com/osquery/osquery/pull/5684))
+
+Don't return a battery row, if there are no results ([#5650](https://github.com/osquery/osquery/pull/5650)) 
+
+Fix several integer conversions in process_ops ([#5614](https://github.com/osquery/osquery/pull/5614))
+
+Include weekends on the kernel_panics table ([#5298](https://github.com/osquery/osquery/pull/5298))
+
+Fix `key_strength` bug for windows certificates table ([#5304](https://github.com/osquery/osquery/pull/5304))
+
+Fix: `interface` column of `routes` table could be empty on Windows ([bcf0ab8e](https://github.com/osquery/osquery/commit/bcf0ab8e))
+
+Fix: `name` column of `programs` table could be empty on Windows ([7bceba4b](https://github.com/osquery/osquery/commit/7bceba4b))
+
+Fix `disable_watcher` flag ([08dc11b7](https://github.com/osquery/osquery/commit/08dc11b7))
+
+Fix: populate `path` column correctly in `firefox_addons` table ([#5462](https://github.com/osquery/osquery/pull/5462))
+
+Fix numeric monitoring plugin not being registered ([#5484](https://github.com/osquery/osquery/pull/5484))
+
+Fix wrong error code returned when querying the Windows registry ([#5621](https://github.com/osquery/osquery/pull/5621))
+
+Fix windows/logical_drives boot partition detection ([#5477](https://github.com/osquery/osquery/pull/5477))
+
+Replace sync calls by async ones ([#5606](https://github.com/osquery/osquery/pull/5606))
+
+Fix rocksDB crash ([a31d7582](https://github.com/osquery/osquery/commit/a31d7582))
+
+Fix bug in table column data validator ([e3037331](https://github.com/osquery/osquery/commit/e3037331))
+
+Fix random port problem ([a32ed7c4](https://github.com/osquery/osquery/commit/a32ed7c4))
+
+Refactor battery table and return some information even if advanced information is missing ([6a64e353](https://github.com/osquery/osquery/commit/6a64e353))
+
+
+### Table Changes
+
+
+Added table `ibridge_info` on macOS (Notebooks only) ([#5707](https://github.com/osquery/osquery/pull/5707))
+
```

Comment:
Sorry for my lack of understanding. Is double-spacing a requirement / intended?

Reply:
Thanks for catching that, totally unintentional, fixing.